plugins/system/guidedtours/src/Extension/GuidedTours.php

```diff
@@ -186,7 +189,7 @@ private function getTour($tourId)
     /**
      * Return a tour and its steps or null if not found
      *
-     * @param   TourTable  $item  The tour to load
+     * @param   CMSObject  $item  The tour to load
```

Comment:
```suggestion
     * @param   \stdClass  $item  The tour to load
```

I think from the code?

Reply:
@wilsonge But the `$item` here is a result of `Joomla\Component\Guidedtours\Administrator\Model::getItem()` which returns `CMSObject`

Reply:
Oh i'm just blind looking at the return type instead of the param 🤦‍♂️